src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -66,11 +69,16 @@ class OppgaveService(
                             iverksett.gjeldendeVedtak())
                 } ?: finnBeskrivelseForFørstegangsbehandlingAvVedtaksresultat(iverksett)
             }
-            Vedtaksresultat.OPPHØRT -> beskrivelseRevurderingOpphørt(iverksett.vedtak.vedtakstidspunkt)
+            Vedtaksresultat.OPPHØRT -> beskrivelseRevurderingOpphørt(opphørstato(iverksett))
             else -> error("Kunne ikke finne riktig vedtaksresultat for oppfølgingsoppgave")
         }
     }
 
+    private fun opphørstato(iverksett: Iverksett): LocalDate {
+        val tilkjentYtelse = iverksett.vedtak.tilkjentYtelse ?: error("TilkjentYtelse er null")
+        return tilkjentYtelse.andelerTilkjentYtelse.maxOf { it.tilOgMed }
```

Comment:
Må kanskje ta høyde for at andelene er tomme - usikker på hva beskrivelsen bør bli isåfall, men tror denne funksjonen bør kunne returnere `LocalDate?` (n